packages/core-mobile/app/new/features/track/components/TokenDetailChart.tsx

```diff
@@ -44,13 +47,33 @@ export const TokenDetailChart = ({
     dispatch(setViewOnce(ViewOnceKey.CHART_INTERACTION))
   }
 
+  const chartLabels = useMemo(() => {
+    const min = Math.min(...(chartData?.map(d => d.value) ?? []))
+    const max = Math.max(...(chartData?.map(d => d.value) ?? []))
```

Comment:
it looks like we already have the min and max values in `ranges` from the `useTokenDetails` hook. should we just pass it to this component so we don't have to recompute?